Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -616,6 +616,45 @@ void PreBattleCalcAuraEffect(struct BattleUnit * attacker, struct BattleUnit * d
                     attacker->battleAttack  += 3;
 #endif
 
+#if (defined(SID_DriveStr) && (SID_DriveStr < MAX_SKILL_NUM))
+                if (SkillTester(unit, SID_DriveStr) && range2[i] == 1)
+				{
```

Comment:
Fix all indentation issues

Reply:
Something weird is going on, the indentation level on Github does not match what I have locally. I manually updated the file on Github, but I have no idea what causing the difference